ci-operator/jobs/openshift/knative-serving/openshift-knative-serving-release-v0.8.1-periodics.yaml

```diff
@@ -0,0 +1,63 @@
+periodics:
+- agent: kubernetes
+  decorate: true
+  extra_refs:
+  - base_ref: release-v0.8.1
+    org: openshift
+    repo: knative-serving
+  interval: 12h
+  name: periodic-ci-openshift-knative-serving-release-v0.8.1-e2e-vsphere-ocp-42
+  spec:
+    containers:
+    - args:
+      - --artifact-dir=$(ARTIFACTS)
+      - --give-pr-author-access-to-namespace=true
```

Comment:
This line probably doesn't make sense.

Reply:
Copied from the openshift release periodics. I'd leave it in for good measure though I agree that it technically doesn't make a lot of sense.